src/components/HeaderCell.js

```diff
@@ -13,13 +12,22 @@ export default class HeaderCell extends React.Component {
 
   render() {
     const { rowIndex, field, data, type, ...props } = this.props
+    if (field !== 'name') {
+      return (
+        <Cell
+          {...props}
+          onClick={this.props.onClick.bind(this, data[rowIndex], type)}
+          className="clickable"
+        >
+          { data[rowIndex][field] }
+        </Cell>
+      )
+    }
```

Comment:
Normally you would avoid having a component with 2 different renders and would opt into conditionally rendering 2 components. Helps keep the UI declarative and clear